Doesn't compile with default-features = false

Hey!

When I build the latest version with "meval = {version = "*", default-features = false}" I get the following error:
```
error[E0432]: unresolved import `serde::de`
 --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/meval-0.0.8/src/de.rs:2:5
  |
2 | use serde::de;
  |     ^^^^^^^^^ Maybe a missing `extern crate serde;`?

error[E0432]: unresolved import `serde::de::Error`
 --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/meval-0.0.8/src/de.rs:3:5
  |
3 | use serde::de::Error;
  |     ^^^^^^^^^^^^^^^^ Maybe a missing `extern crate serde;`?

error[E0432]: unresolved import `serde::Deserialize`
 --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/meval-0.0.8/src/de.rs:4:5
  |
4 | use serde::Deserialize;
  |     ^^^^^^^^^^^^^^^^^^ Maybe a missing `extern crate serde;`?
```

This is not supposed to happen, right?

--- a/.travis.yml
+++ b/.travis.yml
@@ -18,6 +18,7 @@ script:
   travis-cargo test &&
   travis-cargo bench &&
   travis-cargo --only stable doc
+- cargo build --no-default-features --verbose
 after_success:
 - travis-cargo --only stable doc-upload
 env:

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -9,7 +9,7 @@ name = "meval"
 readme = "README.md"
 repository = "https://github.com/rekka/meval-rs"
 version = "0.0.8"
-exclude = ["README.tpl"]
+exclude = ["README.tpl", ".travis.yml"]
 
 [dependencies]
 fnv = "1.0.5"

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -136,7 +136,7 @@
 //! # Deserialization
 //!
 //! [`Expr`][Expr] supports deserialization using the [serde] library to make flexible
-//! configuration easy to set up.
+//! configuration easy to set up, if the feature `serde` is enabled (default).
 //!
 //! ```rust
 //! #[macro_use]
@@ -197,6 +197,8 @@ use std::fmt;
 pub mod tokenizer;
 pub mod shunting_yard;
 mod expr;
+
+#[cfg(feature = "serde")]
 pub mod de;
 
 pub use expr::*;

